unitycatalog-ai/integrations/crewai/pyproject.toml

```
@@ -0,0 +1,68 @@
+[project]
+name = "ucai-crewai"
+version = "0.0.1"
+description = "Support for Unity Catalog functions as CrewAI tools"
+authors = [
+    { name="Michael Berk", email="michaelberk99@gmail.com" }
+]
+readme = "README.md"
+license = { text="MIT" }
+requires-python = ">=3.10"
+dependencies = [
+    "ucai-core",
+    "crewai>=0.67.1",
+    "crewai_tools",
+]
+
+[project.optional-dependencies]
+dev = [
+  "pytest",
+  "typing_extensions",
+  # temporarily needed until ucai-core[databricks] is released
+  "databricks-sdk>=0.32.0",
+  "databricks-connect"
+]
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build]
+include = [
+  "src/ucai_crewai/*"
```

```suggestion
  "src/unitycatalog/ai/crewai/*"
```
FYI for refactoring (this is the target path that needs to be created)